docs/open-source/_toc.json

```
@@ -39,6 +39,15 @@
           "url": "https://github.com/Qiskit/qiskit/blob/main/MAINTAINING.md"
         }
       ]
+    },
+    {
+      "title": "Creating a Qiskit-based project",
```

```suggestion
      "title": "Create a Qiskit-based project",
```
Sometimes it feels awkward to do so but we try to avoid -ing in titles for consistency and to clearly suggest action